### PR TITLE
add info about hipchat notify param

### PR DESCRIPTION
**Pull request by porrl**

### CODE REVIEW

File: docs/user/alert-definitions.rst [hunk 1]
```diff
@@ -144,7 +144,7 @@ The notifications field is a list of function calls (see below for examples), ca
 .. py:function:: send_sms(number*, [message, repeat])
 .. py:function:: send_push([message, repeat, url, key])
 .. py:function:: send_slack([channel, message, repeat, token])
-.. py:function:: send_hipchat([room, message, color='red', repeat, token])
+.. py:function:: send_hipchat([room, message, color='red', repeat, token, notify='False'])
```
Comment (hjacobs) [inline docs/user/alert-definitions.rst:147]: 'False" in quotes? should be a boolean not a string I guess...

File: docs/user/alert-definitions.rst [hunk 2]
```diff
@@ -192,7 +192,8 @@ Example JSON HipChat configuration:
    [
       "send_hipchat()",
       "send_hipchat(room='#incidents', color='red')",
-      "send_hipchat(room='#incidents', token='your-token')"
+      "send_hipchat(room='#incidents', token='your-token')",
+      "send_hipchat(room='#incidents', token='your-token', notify='True')"
```
Comment (hjacobs) [inline docs/user/alert-definitions.rst:196]: same here: should be True without quotes (I assume)

Comment (porrl) [inline docs/user/alert-definitions.rst:196]: Without quotes it throws: 
zmon_worker_monitor/zmon_worker/tasks/notacelery_task.py", line 946, in safe_eval 
NameError: name 'True' is not defined

Quotes make it work.

Comment (Jan-M) [inline docs/user/alert-definitions.rst:196]: I added True and False to the context in the worker.

**Pull request merged**